src/app/app-modules/nurse-doctor/case-sheet/general-case-sheet/doctor-diagnosis-case-sheet/doctor-diagnosis-case-sheet.component.css

```
@@ -1,112 +1,531 @@
+/* Textarea Fields */
+.textarea {
+    min-height: 80px;
+    border: 1px solid #ced4da;
+    border-radius: 4px;
+    padding: 12px;
+    background: white;
+    font-size: 13px;
+    line-height: 1.5;
+    color: #212529;
+    white-space: pre-wrap;
+    word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)